scss/win/elements/_itemBox.scss

```diff
@@ -17,10 +17,21 @@ item-box {
 		margin-bottom: 0;
 	}
 	
-	input {
-		padding: 2px;
-		margin: -1px 0;
-		margin-inline-start: 1px;
+	@media (-moz-platform: windows-win10) {
+		td > input, td > textarea, .creator-name-box > input {
+			// Give text fields a consistent, native-ish look using Windows 11
+			// colors. Ideally, we only want to do this on Windows 11, but
+			// Windows 11 can't currently be specifically targeted.
+			// https://bugzilla.mozilla.org/show_bug.cgi?id=1825447
+			padding: 2px 3px;
+			border: 1px solid #838383;
```

Comment:
I think for colours like these we should use either mozilla's own defined `--toolbar-field-border-color` or `ThreeDShadow` css variables if they match our wanted look. That way when we add support for Dark Mode, we will have less work.

Reply:
Yeah, that gray looks fine. I guess `var(--toolbar-field-border-color)`, to be less weird Mozilla-y.

Reply:
Huh! OK, well, still, probably better to go with the CSS variable than something deprecated.